e-learning-webapps/src/main/webapp/vue-app/main.js

```diff
@@ -11,7 +12,7 @@ const lang = eXo && eXo.env && eXo.env.portal && eXo.env.portal.language || 'en'
 
 const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.addon.elearning-${lang}.json`;
 
-
+export const EventBus = new Vue();
```

Comment:
Why is it called EventBus ?

Reply:
I couldn't use just "Event" so I thought it would be a unique name, should I change it to "exoEvents" or something like that ?